RULES.md

```diff
@@ -67,12 +67,12 @@ The [AWS Solutions Library](https://aws.amazon.com/solutions/) offers a collecti
 | AwsSolutions-EFS1  | The EFS is not configured for encryption at rest.                                                                                                                                         | By using an encrypted file system, data and metadata are automatically encrypted before being written to the file system. Similarly, as data and metadata are read, they are automatically decrypted before being presented to the application. These processes are handled transparently by EFS without requiring modification of applications.                                                                                                                                                                                                                       |
-| AwsSolutions-ELB1  | The ELB is used for incoming HTTP/HTTPS traffic. Use ALBs instead.                                                                                                                        | HTTP/HTTPS applications (monolithic or containerized) should use the ALB instead of the ELB for enhanced incoming traffic distribution, better performance and lower costs.                                                                                                                                                                                                                                                                                                                                                                                            |
+| AwsSolutions-ELB1  | The CLB is used for incoming HTTP/HTTPS traffic. Use ALBs instead.                                                                                                                        | HTTP/HTTPS applications (monolithic or containerized) should use the ALB instead of The CLB for enhanced incoming traffic distribution, better performance and lower costs.                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
```suggestion
| AwsSolutions-ELB1  | The CLB is used for incoming HTTP/HTTPS traffic. Use ALBs instead.                                                                                                                        | HTTP/HTTPS applications (monolithic or containerized) should use the ALB instead of the CLB for enhanced incoming traffic distribution, better performance and lower costs.                                                                                                                                                                                                                                                                                                                                                                                            |
```